src/components/Settings/ChangePasswordForm.js

```diff
@@ -29,26 +29,22 @@ const ChangePasswordForm = () => {
   };
 
   return (
-    <Form
-      formMethods={formMethods}
-      className={styles.settingsForm}
-      onSubmit={onSubmit}
-    >
+    <Form className={formStyles} formMethods={formMethods} onSubmit={onSubmit}>
       <Form.Input
         label={intl.messages['common.newPassword']}
         name="password"
         type="password"
         data-testid="password-input-settings"
       />
       <Form.Button
+        className={buttonStyles}
```

Comment:
Shouldn't you be using `styles` instead of `className`? Or did I got confused?

Reply:
yeah, I forgot that one 👀